docs/index.md

```diff
@@ -4,204 +4,119 @@ title: "Introduction to ZIO Logging"
 - ZIO Console, SLF4j, and other backends
 
 ## Installation
 
-`ZIO-Logging` is available via maven repo.
-In order to use this library, we need to add the following line in our build.sbt file:
+In order to use this library, we need to add the following line in our `build.sbt` file:
 
 ```scala
-libraryDependencies += "dev.zio" %% "zio-logging" % @VERSION@
+libraryDependencies += "dev.zio" %% "zio-logging" % "@VERSION@"
 ```
 
-### Log Format
-
-A `LogFormat` represents a DSL to describe the format of text log messages.
+There are also some optional dependencies:
 
 ```scala
-import zio.logging.console
-import zio.logging.LogFormat._
-
-val myLogFormat = timestamp.fixed(32) |-| level |-| label("message", quoted(line))
-val myConsoleLogger = console(myLogFormat)
-```
-
-### Logger Context and Annotations
-
-The `logContext` fiber reference is used to store typed, structured log
-annotations, which can be utilized by backends to enrich log messages.
+// SLF4j integration
+libraryDependencies += "dev.zio" %% "zio-logging-slf4j" % "@VERSION@"
 
-Because `logContext` is an ordinary `zio.FiberRef`, it may be get, set,
-and updated like any other fiber reference. However, the idiomatic way to
-interact with `logContext` is by using `zio.logging.LogAnnotation`.
+// Using ZIO Logging for SLF4j loggers, usually third-party non-ZIO libraries
+libraryDependencies += "dev.zio" %% "zio-logging-slf4j-bridge" % "@VERSION@"
 
-For example:
+// Scala.js console integration
+libraryDependencies += "dev.zio" %% "zio-logging-jsconsole" % "@VERSION@"
 
-```scala
-myResponseHandler(request) @@ LogAnnotation.UserId(request.userId)
+// Scala.js HTTP Logger which sends logs to a backend via Ajax POST
+libraryDependencies += "dev.zio" %% "zio-logging-jshttp" % "@VERSION@"
 ```
 
-This code would add the structured log annotation `LogAnnotation.UserId`
-to all log messages emitted by the `myResponseHandler(request)` effect.
+## Example
 
-The user of the library is allowed to add a custom `LogAnnotation`:
+Let's try an example of ZIO Logging which demonstrates a simple application of ZIO logging along with its _Logger Context_ feature:
 
-```scala
-import zio.logging.LogAnnotation
-
-val customLogAnnotation = LogAnnotation[Int]("custom_annotation", _ + _, _.toString)
-```
-
-### Logger setup in ZIO application
-
-The recommended place for setting the logger is application boostrap.
-In this case, custom logger will be set for whole application runtime (also application failures will be logged with specified logger).
+[//]: # (TODO: make snippet type-checked using mdoc)
 
 ```scala
-package zio.logging.example
+import zio.clock.Clock
+import zio.duration.durationInt
+import zio.logging._
+import zio.random.Random
+import zio.{ExitCode, NonEmptyChunk, ZIO}
 
-import zio.logging.{ LogFormat, console }
-import zio.{ ExitCode, Runtime, Scope, ZIO, ZIOAppArgs, ZIOAppDefault, ZLayer }
+object ZIOLoggingExample extends zio.App {
```

Comment:
this look like zio-logging version 1 example

Reply:
Done!